windows/internal/vs2019_install.ps1

```diff
@@ -0,0 +1,21 @@
+$VS_DOWNLOAD_LINK = "https://aka.ms/vs/16/release/vs_buildtools.exe"
+$VS_INSTALL_ARGS = @("--nocache","--quiet","--wait", "--add Microsoft.VisualStudio.Workload.VCTools",
+                                                     "--add Microsoft.Component.MSBuild",
+                                                     "--add Microsoft.VisualStudio.Component.Roslyn.Compiler",
+                                                     "--add Microsoft.VisualStudio.Component.VC.CoreBuildTools",
+                                                     "--add Microsoft.VisualStudio.Component.VC.Redist.14.Latest",
+                                                     "--add Microsoft.VisualStudio.Component.VC.Tools.x86.x64")
+
+curl.exe --retry 3 -kL $VS_DOWNLOAD_LINK --output vs_installer.exe
+if ($LASTEXITCODE -ne 0) {
+    echo "Download of the VS 2019 installer failed"
+    exit 1
+}
+
+$process = Start-Process "${PWD}\vs_installer.exe" -ArgumentList $VS_INSTALL_ARGS -NoNewWindow -Wait -PassThru
+Remove-Item -Path vs_installer.exe -Force
+$exitCode = $process.ExitCode
+if (($exitCode -ne 0) -and ($exitCode -ne 3010)) {
```

Comment:
Can you please copy VSCollect.exe logic here from https://github.com/pytorch/pytorch/blob/master/.circleci/scripts/vs_install.ps1#L25

And this script is not at all used right now, isn't it?

Reply:
Used https://github.com/pytorch/builder/pull/445/files#diff-c51838f8d7c4e93c825b6382dfc0f40dR2 and https://github.com/pytorch/builder/pull/445/files#diff-e168284874ac9b62e0ccaeba8f6b5db1R2.

Reply:
> Can you please copy VSCollect.exe logic here from https://github.com/pytorch/pytorch/blob/master/.circleci/scripts/vs_install.ps1#L25

Sound reasonable. Will need to upload them as artifacts in binary jobs,